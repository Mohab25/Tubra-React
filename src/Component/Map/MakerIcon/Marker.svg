<svg width="800" height="600" xmlns="http://www.w3.org/2000/svg">

 <g>
  <title>Layer 1</title>
  <ellipse stroke-width="0" ry="150" rx="150" id="svg_1" cy="295" cx="400" opacity="undefined" stroke="#000" fill="#ffff08"/>
  <text transform="matrix(1,0,0,1,0,0) " xml:space="preserve" text-anchor="start" font-family="sans-serif" font-size="100" id="svg_2" y="320.5" x="363.89063" opacity="undefined" stroke-opacity="null" stroke-dasharray="null" stroke-width="0" stroke="#000" fill="#ffffff">C</text>
 </g>
</svg>