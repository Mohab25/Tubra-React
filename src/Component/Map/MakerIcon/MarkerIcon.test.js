import React from 'react'
import Marker from './MarkerIcon'

describe('render tests',()=>{
    it('renders correctly - snapshot test',()=>{
        expect(<Marker/>).toMatchSnapshot()
    })
})