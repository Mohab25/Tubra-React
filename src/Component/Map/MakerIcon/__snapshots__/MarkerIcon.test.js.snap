// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render tests renders correctly - snapshot test 1`] = `<MarkerIcon />`;
